R/tidyverse.R

```diff
@@ -110,3 +113,22 @@ use_tidy_eval <- function() {
 
   todo("Run document()")
 }
+
+
+#' @export
+#' @rdname tidyverse
+#' @inheritParams use_template
+use_tidy_contributing <- function() {
+  check_uses_travis()
+
+  name <- project_name()
+  gh <- gh::gh_tree_remote(base_path)
```

Comment:
What is `base_path` here? It's causing a new NOTE re: `no visible binding for global variable`.

Reply:
Oh, shoot— sorry, one sec.